project/Dependencies.scala

```diff
@@ -9,7 +9,8 @@ object Dependencies {
 
   lazy val authUtils = "uk.gov.nationalarchives" %% "tdr-auth-utils" % "0.0.99"
   lazy val generatedGraphql = "uk.gov.nationalarchives" %% "tdr-generated-graphql" % "0.0.282"
-  lazy val awsUtils =  "uk.gov.nationalarchives" %% "tdr-aws-utils" % "0.1.52"
+  lazy val s3Utils =  "uk.gov.nationalarchives" %% "s3-utils" % "0.1.55"
```

Comment:
Might be worth putting the version in a `private val`

Reply:
I've added a `val` into this one.